packages/components/package.json

```diff
@@ -75,6 +85,7 @@
     "@glint/template": "^1.4.0",
     "@rollup/plugin-babel": "^6.0.4",
     "@tsconfig/ember": "^3.0.8",
+    "@types/codemirror": "^5.60.15",
```

Comment:
this should not be required (`@codemirror/state` and `@codemirror/view` have their type declarations in the packages)